src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -250,6 +273,10 @@ export default {
 			return true
 		},
 
+		isClickable() {
+			return !!this.$listeners?.click || !this.popoverEmpty
+		},
+
```

Comment:
This is not portable to Vue 3. It may work if we remove `click` from component's events, but by design component is not aware of its event subscribers.

It also should not be clickable when it is just a `div`, otherwise there is an a11y issue. So we can check for `popoverEmpty` and `isLinkComponent`.

Reply:
This is still an issue. For links interactive styles may be applied when it's `<a>`.

Reply:
I thought I dropped this commit